generate_testVectors_json.py

```diff
@@ -806,9 +804,9 @@ def unhex(a):
     "reduced": {
         "manifest": protobuf_to_dict(current_manifest),
         "keycards": {
-            kc1.address: hex(newKc1.enroll_keycard.user_wallet.raw),
-            kc2.address: hex(newKc2.enroll_keycard.user_wallet.raw),
-            guestKeyPair.address: hex(zero_addr.raw),
+            hex(newKc1.enroll_keycard.user_wallet.raw):kc1.address,
+            hex(newKc2.enroll_keycard.user_wallet.raw):kc2.address,
+            hex(zero_addr.raw):guestKeyPair.address,
```

Comment:
how are you dealing with multiple guests then? this will be the case for clerks at least.

Reply:
you mean multiple keycards to one wallet address?
the only purpose we are currently using the public keys are for event verification. We dont necessarily need to get all the KC associated with a WA for now.

Reply:
no, i mean: i you key them by the zero address, two guests will collide, potentially overriding the previous guest.

Reply:
if we really _only_ store these for event verification we might as well make them a big list/set

Reply:
wait.. im getting confused by these names now. what i want is KC=>WA which its already doing before this change right?